README.md

```diff
@@ -12,3 +12,8 @@
 ## Features
 
 - Create and edit Kaoto (`*.kaoto.yaml` and `*.kaoto.yml`) files.
+
+## Data and Telemetry
+
+The VS Code Kaoto extension collects anonymous [usage data](USAGE_DATA.md) and sends it to Red Hat servers to help improve our products and services. Read our [privacy statement](https://developers.redhat.com/article/tool-data-collection) to learn more. This extension respects the `redhat.telemetry.enabled` setting which you can learn more about at https://github.com/redhat-developer/vscode-redhat-telemetry#how-to-disable-telemetry-reporting
+>>>>>>> 49c5b92 feat: Include opt-in Red Hat telemetry
```

Comment:
Did this comment sneak in here by mistake or am I seeing things?

Reply:
mistake. I will fix it